server/config.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
     scriptCache: '24h',
     matchingRecordsToBeShownPerPage: 5,
     maximumPages: 4,
+    minimumMatchProbability: 0.5,
```

Comment:
why has this come back? I thought it had just been removed

Reply:
In the ticket I missed the fact that this still needs to be there except that we remove this filtering when clicking on Show more records